prac_05/emails.py

```diff
@@ -0,0 +1,67 @@
+"""
+CP1404 2022 Prac 05
+Erica Finlay
+
+Do-From_Scratch Exercises
+Emails
+
+Estimate:   2 hours
+Actual:     1 hour 10 min
+
+The solution to this program was consulted on GitHub.
+
+Pseudocode:
+function main()
+    email_and_user = empty dictionary
+    length_of_name = 0
+    get email
+    while email != ""
+        get_name_from_email(email)
+        get confirmation
+        if confirmation != "Y" and confirmation != ""
+            get name
+        add name as value
+        for email in email_and_user
+            if length of name > length_of_name
+                length_of_name = length of name
+        get email
+    for email, name in email_and_user
+        display name, email
+
+
+function get_name_from_email(email)
+    initial_string = split email at '@' [0]
+    names = split initial_string at '.'
+    name = join names with space between
+    return name
+"""
+
+
+def main():
+    """Ask the user for their email and name and prints this information."""
```

Comment:
Docstrings use imperative voice ('This function will..') so perhaps it say 'print', rather than 'prints'

Reply:
Thanks Kath - that was a really clever one to pick up.  Much appreciated!